TESTING.md

```diff
@@ -25,6 +25,59 @@ template (`catalogItem`) are defined in the configuration file, new ones will be
 the test. You can choose to preserve catalog and vApp template across runs (use the `preserve` field in the
 configuration file).
 
+The tests can run with several tags that define which components are tested.
+Using the Makefile, you can run one of the following:
+
+```bash
+make testcatalog
+make testnetwork
+make testgateway
+make testvapp
+make testvm
+```
+
+For more options, you can run manually in `./vcd`
+When running `go test` without tags, we'll get a list of tags that are available.
+
+```bash
+$ go test -v .
+=== RUN   TestTags
+--- FAIL: TestTags (0.00s)
+    api_test.go:66: # No tags were defined
+    api_test.go:41:
+        # -----------------------------------------------------
+        # Tags are required to run the tests
+        # -----------------------------------------------------
+
+        At least one of the following tags should be defined:
+
+           * ALL :       Runs all the tests
+           * functional: Runs all the acceptance tests
```

Comment:
Empty line above would be nice to differentiate ALL from other two set types.

Reply:
done